### ISSUE TITLE
Redis: Option to use plain Strings instead of Hashes.

### ISSUE THREAD

**Issue by rizlik** [brianfrankcooper/YCSB#1097]
YCSB benchmark against Redis uses Hashes (HGET/HSET commands) instead of plain Redis String (GET/SET commands).

I think that it would be good to have an option to use Plain String because in some contexts performances differ if using plain String or Hashes.

**Comment by busbey** [comment]
closes #1129

**Comment by photoszzt** [comment]
Is there any progress after two years? Looking at the code, it's still using Hashes.

**Comment by busbey** [comment]
IIRC #1129 addresses this feature request? The quickest way to help get this issue solved is to review PRs.

**Comment by photoszzt** [comment]
Yes. I'm a new user of YCSB and I'm not familair with the issue the #1129 author is talking about and block on.

**Comment by rizlik** [comment]
This was part of some work done with a project ended long ago, I'm not sure I can help too much on this. 

> Yes. I'm a new user of YCSB and I'm not familair with the issue the #1129 author is talking about and block on.

So #1129 store the object as a single JSON object represented as a string blob. As far as I remember the problem with this approach is that you cannot selectively delete one single fields without reading (and storing back) the full object. AFAIR this violates the semantic of some YCSB commands.

**Comment by photoszzt** [comment]
I assume Memcached also has this problem? How does that platform solve the issue?
